<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionDialog</class>
 <widget class="QDialog" name="ConnectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>512</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>453</width>
    <height>512</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>453</width>
    <height>512</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>连接数据库</string>
  </property>
  <widget class="QLabel" name="label_db">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>35</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>数据库类型：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_hostName">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>95</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>主机名或IP地址：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_hostName">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>127.0.0.1</string>
   </property>
   <property name="maxLength">
    <number>15</number>
   </property>
   <property name="cursorPosition">
    <number>9</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_db">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>SQL SERVER</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MYSQL</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>其他</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_port">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>185</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>端口：</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_port">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>180</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="maximum">
    <number>65555</number>
   </property>
   <property name="value">
    <number>1433</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_dbName">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>255</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>数据库名称：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_uid">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>用户名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_pwd">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>370</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_pwd">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>380</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_uid">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>320</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string>sa</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_dbName">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>250</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SalaryDMS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_conn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>460</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>开始连接</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancle">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>460</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
